assembly/assembly-wsmaster-war/src/main/java/org/eclipse/che/api/deploy/CheWebSocketEndpoint.java

```diff
@@ -0,0 +1,36 @@
+/*******************************************************************************
```

Comment:
Can we use the same class https://github.com/eclipse/che/blob/master/assembly/assembly-wsagent-war/src/main/java/org/eclipse/che/wsagent/server/CheWebSocketEndpoint.java ?